docs/tutorial/code/django/task.yaml

```diff
@@ -7,7 +7,13 @@
 ###########################################
 summary: Getting started with Django tutorial
 
-kill-timeout: 180m
+kill-timeout: 50m
+
+restore: |
+  rm -f $HOME/*.yaml $HOME/*.py $HOME/*.txt
```

Comment:
Could put everything in a subdirectory of `$HOME` for each of these tasks so we can just delete that subdirectory?